static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -59,9 +59,15 @@ function ActionSet({
 
   const label = getLabel(numIssues, allInQuerySelected);
 
-  const selectedIssues = [...issues]
-    .map(issueId => GroupStore.get(issueId))
-    .filter(issue => issue) as BaseGroup[];
+  const selectedIssues: BaseGroup[] = [];
+  for (const issue of issues) {
+    const resolvedIssue = GroupStore.get(issue);
+
+    if (resolvedIssue) {
+      // @TODO use a type guard here
+      selectedIssues.push(resolvedIssue as BaseGroup);
+    }
+  }
```

Comment:
this feels harder to read

Reply:
could just switch it to issues.map so there isn't an extra array being made

Reply:
Actually, set.forEach is better here, same with below comment I think